packages/shopping/src/controllers/specs/user-controller.specs.ts

```diff
@@ -41,3 +41,22 @@ export const CredentialsRequestBody = {
     'application/json': {schema: CredentialsSchema},
   },
 };
+
+const RefreshTokenSchema = {
+  type: 'object',
+  required: ['refreshToken'],
+  properties: {
+    refreshtoken: {
+      type: 'string',
+      minLength: 8,
```

Comment:
> minLength: 8

Usually the jwt tokens we generate are very long. What is the reasoning for 8? Why not 1? Or why specify this at all? Just curious

Reply:
same question

Reply:
@emonddr If you look at the `generateToken` function in
https://github.com/strongloop/loopback4-example-shopping/pull/537/files#diff-90f4ee64422c3d610fc44a185e6d0781R47

A refresh token is the id of a `UserRefreshToken` record, not a token generated by jwt service.

Reply:
I would prefer a pattern check to the token, but i'm still not sure what is the best way to generate a `refreshToken`. Currently it's a `objectId` which might not be the best practice. On a `mysql`/`postgresql` database i would just use a UUID as `refreshToken` and use a regular expression for validation.

Do you have any other ideas how to generate  a revokeable token?